spec/spec_helper.rb

```diff
@@ -1,5 +1,6 @@
 require 'simplecov'
 SimpleCov.start
+require 'capybara/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.